yarn-project/prover-client/src/prover-agent/prover-agent.ts

```diff
@@ -121,15 +121,15 @@ export class ProverAgent {
 
   private async work(jobSource: ProvingJobSource, job: ProvingJob<ProvingRequest>): Promise<void> {
     try {
-      this.log.info(`Picked up proving job id=${job.id} type=${ProvingRequestType[job.request.type]}`);
+      this.log.debug(`Picked up proving job id=${job.id} type=${ProvingRequestType[job.request.type]}`);
```

Comment:
Oh we actually wanted this for the deployed nodes, but now we have it print out its status every 60 seconds so we can go back to making it `debug`